core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala

```diff
@@ -311,7 +311,8 @@ case class SnipDirective(page: Page, variables: Map[String, String])
         } else new File(page.file.getParentFile, source)
       val text = Snippet(file, labels)
       val lang = Option(node.attributes.value("type")).getOrElse(Snippet.language(file))
-      new VerbatimNode(text, lang).accept(visitor)
+      val group = Option(node.attributes.value("group")).getOrElse("")
```

Comment:
What do you think about the idea mentioned here: https://github.com/lightbend/paradox/pull/95#discussion_r100283649

That would allow easier adaptation of the group functionality. All of the already written docs could use this feature without any changes to the docs.